### PR TITLE
WIP Add django 2.2 support.

### LINKED ISSUE #848: Django 2.2 upgrade
More of a long-term goal, but decided to have a look at what would be required to get there. Some of the modules used don't currently support Django 2.0:

* django-cms
    * Planned for the 3.6 release, no date scheduled.
* django-authority
    * Unmaintained.
* django-filer
    * PR fixing waiting to be merged: https://github.com/divio/django-filer/pull/1096

In the meantime, the deprecation warnings coming from the application itself can be fixed.

### PR DESCRIPTION
**Pull request by umgelurgel**
fix #848 

WIP for reasons mentioned in the ticket - models depending on packages that have no support will have to be retired first.

### CODE REVIEW

**Comment by artcz** [comment]
whoa :o
Looks great, I'm patiently waiting for where this is going next. :)

File: requirements.in [hunk 1]
```diff
@@ -11,8 +11,10 @@ django-crispy-forms
 
 # we can probably remove tagging - taggit is used with custom models for cfp
 django-tagging==0.4.6
-django-taggit==0.22.1
-django-taggit-labels==0.6.0
+django-taggit
```
Comment (malemburg) [inline requirements.in]: Why are you removing the version pinning ?

File: requirements.in [hunk 2]
```diff
@@ -22,17 +24,12 @@ social-auth-core[openidconnect]
 
 markdown2
 
-django-cms>=3.6,<3.7
+django-cms
```
Comment (malemburg) [inline requirements.in]: Same note as above.

**Comment by malemburg** [comment]
There are still several warnings caused by the build: https://travis-ci.org/EuroPython/epcon/builds/641998515#L1066

Most of them seem to be datetime object related.

**Comment by malemburg** [comment]
Most of the changes in the PR are simple changes. Is that really all that's needed to upgrade to Django 2.2 ?

I'm somewhat worried about things which may break, which we don't have on the radar now, one week before starting to prepare the website for go-live.

File: requirements.in (hunk 1 above)
Comment (matrixise) [reply requirements.in]: It's `django-taggit==1.2.0`

**Comment by aramboi** [comment]
Last commit should also Fix #1181

**Comment by aramboi** [comment]
Closing in favor of https://github.com/EuroPython/epcon/pull/1241

**Pull request closed without merging**